### PR TITLE
Docs | Switch to new docs for Microsoft.Data.SqlClient

### PR DESCRIPTION
**Pull request by cheenamalhotra**
Lightweight PR to switch docs from .NET Framework to Sql Connect > ADO.NET (recently added)
Major path changes:

Old URL (/dotnet/framework/data/adonet/**) | NEW URL (/sql/connect/ado-net/**)
-- | --
query-notifications-in-sql-server | query-notifications-sql-server
transaction-and-bulk-copy-operations | transaction-bulk-copy-operations
ado-net-overview | overview-sqlclient-driver
configuring-parameters-and-parameter-data-types | configure-parameters
generating-commands-with-commandbuilders | generate-commands-with-commandbuilders
retrieving-and-modifying-data | retrieving-modifying-data
sqlclient-support-for-localdb | sqlclient-support-localdb
sql/sqlclient-support-for-high-availability-disaster-recovery | sql/sqlclient-support-high-availability-disaster-recovery
adding-existing-constraints-to-a-dataset | add-existing-constraints-to-dataset
commands-and-parameters | commands-parameters
updating-data-sources-with-dataadapters | update-data-sources-with-dataadapters

### CODE REVIEW

File: doc/snippets/Microsoft.Data.SqlClient.Server/SqlMetaData.xml [hunk 1]
```diff
@@ -218,7 +218,7 @@ The following are the default values assigned to `dbType`, depending on the `Sql
                 <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
-For more information, see [Table-Valued Parameters](/dotnet/framework/data/adonet/sql/table-valued-parameters).  
+For more information, see [Table-Valued Parameters](/sql/connect/ado-net/sql/table-valued-parameters).
```
Comment (JRahnama) [inline doc/snippets/Microsoft.Data.SqlClient.Server/SqlMetaData.xml:221]: sql folder is repeated in the path. Is it by design? It has happened in some other places.

**Review by JRahnama — APPROVED** [review]

File: doc/snippets/Microsoft.Data.SqlClient.Server/SqlMetaData.xml (hunk 1 above)
Comment (cheenamalhotra) [reply doc/snippets/Microsoft.Data.SqlClient.Server/SqlMetaData.xml:221]: They were ported as it is from original location

**Review by DavoudEshtehari — APPROVED** [review]
NIT: The reference document (Sql Connect) uses **sentence case style** for titles and would be better to keep it in the same style.

File: doc/snippets/Microsoft.Data.SqlClient/SqlClientPermission.xml [hunk 2]
```diff
@@ -40,7 +40,7 @@ holes in the permission set.
 The <xref:System.Security.Permissions.PermissionState> enumeration takes precedence over the <xref:System.Data.Common.DBDataPermission.AllowBlankPassword%2A> property. 
 Therefore, if you set <xref:System.Data.Common.DBDataPermission.AllowBlankPassword%2A> to `false`, you must also set <xref:System.Security.Permissions.PermissionState> to `None` to 
 prevent a user from making a connection using a blank password. For an example demonstrating how to use security demands, see [Code Access Security and 
-ADO.NET](/dotnet/framework/data/adonet/code-access-security).  
+ADO.NET](/sql/connect/ado-net/code-access-security).
```
[the remark this answers is not in the document]
Comment (cheenamalhotra) [reply doc/snippets/Microsoft.Data.SqlClient/SqlClientPermission.xml:43]:
```suggestion
ADO.NET](/dotnet/framework/data/adonet/code-access-security).  
```

**Pull request merged**